### ISSUE TITLE
host_endian appears to be broken

### ISSUE THREAD

**Issue by haesbaert** [mirage/ocaml-cstruct#72]
```
cstruct bpf_hdr {
    uint32_t     bh_sec;
    uint32_t     bh_usec;
    uint32_t     bh_caplen;
    uint32_t     bh_datalen;
    uint16_t     bh_hdrlen;
} as host_endian
```

Fails to find Cstruct.HE while little_endian and big_endian work fine:
Error: Unbound module Cstruct.HE

OS X with ocaml 4.02.3

**Comment by djs55** [comment]
Interesting, I didn't know this feature was supposed to exist ;) but it's clearly in the camlp4 and ppx.

**Comment by haesbaert** [comment]
Solved in 9569a8a1ccf490d325dbc30a6553bd84d7999c14
